dockerfiles/e2e/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Validate selenium base URL
+if [ -z "$TS_SELENIUM_BASE_URL" ]; then
+    echo "The \"TS_SELENIUM_BASE_URL\" is not set!";
+    echo "Please, set the \"TS_SELENIUM_BASE_URL\" environment variable."
+    exit 1
+fi
+
+# Launch selenium server
+/usr/bin/supervisord --configuration /etc/supervisord.conf & \
+export TS_SELENIUM_REMOTE_DRIVER_URL=http://localhost:4444/wd/hub
+sleep 1
+
+# Check selenium server launching
+expectedStatus=200
+currentTry=1
+maximumAttempts=5
+
+while [ $(curl -s -o /dev/null -w "%{http_code}" --fail http://localhost:4444/wd/hub/status) != $expectedStatus ];
+do
+
+  if (( currentTry > maximumAttempts ));
+  then
+    status=$(curl -s -o /dev/null -w "%{http_code}" --fail http://localhost:4444/wd/hub/status)
+    echo "Exceeded the maximum number of checking attempts,"
+    echo "selenium server status is '$status' and it different to expected '$expectedStatus'";
+    exit 1;
+  fi;
+
+  echo "Wait selenium server availability ..."
+
+  curentTry=$((curentTry + 1))
+  sleep 1
+
+done
+
+
```

Comment:
I see you are installing vnc server in dockerfile, but you are not starting it - is that intentional?

Reply:
I have created an issue for adding this tool https://github.com/eclipse/che/issues/14341